lang/fpc/Portfile

```diff
@@ -136,6 +137,7 @@ if {${subport} eq "${name}"} {
                 -o ${destroot}${fpcbasepath}/etc/fpc.cfg
         "
         ln -s ${fpcbasepath}/etc/fpc.cfg ${destroot}${prefix}/etc
+        system "patch ${destroot}${fpcbasepath}/etc/fpc.cfg ${filespath}/fpc.cfg.patch"
```

Comment:
Why are you not using the `patchfiles` command?

Reply:
If i understood it right, the patch files command is applied before the build. The file, which needs this patch (fpc.cfg) is created during post-destroot with fpcmkcfg. If there is a way to achieve this, i would be glad to know.

Reply:
OK, that makes sense.